lib/src/source_visitor.dart

```diff
@@ -4,8 +4,10 @@
 
 library dart_style.src.source_visitor;
 
-import 'package:analyzer/analyzer.dart';
+import 'package:analyzer/analyzer.dart' show ParameterKind;
```

Comment:
Is the `show` clause needed to avoid collisions in names exported both by this and the imports below? If so, can you leave a comment to that effect. If it isn't needed to avoid a collision, let's go ahead and omit the `show`.